packages/server/__tests__/addFeatureFlag.test.ts

```diff
@@ -0,0 +1,50 @@
+import {sendPublic, signUp} from './common'
+import ServerAuthToken from '../database/types/ServerAuthToken'
+import encodeAuthToken from '../utils/encodeAuthToken'
+
+const ADD_FEATURE_FLAG = `
+  mutation AddFeatureFlag($emails: [String!], $domain: String, $flag: String!) {
+    addFeatureFlag(emails: $emails, domain: $domain, flag: $flag) {
+      error {
+        title
+        message
+      }
+      users {
+        id
+        featureFlags {
+          poker
```

Comment:
I don't have any tension about the test failing, since https://github.com/ParabolInc/parabol/pull/5481 is far from being merged. I will fix it once #5699 is merged.